### PR TITLE
refactor: 💡 fix styling for ordered series diagram

### PR DESCRIPTION
**Pull request by dvonanderson**
## Description
Added wrapper and min-height to `ordered-series-diagram` to ensure when toggles are clicked we don't get weird height behavior.

:technologist: [Admin preview](PATH_TO_FEATURE)

### Screenshots (if appropriate):

Screen Shot 2023-01-25 at 13 30 59

### CODE REVIEW

File: ui/admin/app/styles/app.scss [hunk 1]
```diff
@@ -598,69 +598,71 @@
...
+        text-align: center;
       }
-    }
-  }
 
-  &-group {
-    display: flex;
+      &-title {
+        font-weight: var(--token-typography-font-weight-semibold);
+        margin-top: 0.5rem;
```
Comment (cameronperera) [inline ui/admin/app/styles/app.scss:622]: Should we be using `sizing.rems(xs)` in place of rem directly for these styles?
`_sizing.scss` should have some default sizes already defined.

**Review by cameronperera — CHANGES_REQUESTED** [review]

File: ui/admin/app/styles/app.scss (hunk 1 above)
Comment (dvonanderson) [reply ui/admin/app/styles/app.scss:622]: @cameronperera good catch

File: ui/admin/app/styles/app.scss [hunk 2]
```diff
@@ -598,69 +598,71 @@
...
+.ordered-series-diagram-wrapper {
+  min-height: 150px;
 
-  width: 100%;
-  display: flex;
-  margin-bottom: 1rem;
-
-  &-item {
+  .ordered-series-diagram {
+    --group-title-line-height: 2.25rem;
```
Comment (cameronperera) [inline ui/admin/app/styles/app.scss:605]:
```suggestion
    --group-title-line-height: sizing.rems(xl);
```

File: ui/admin/app/styles/app.scss [hunk 3]
```diff
@@ -598,69 +598,71 @@
...
       }
-    }
-  }
 
-  &-group {
-    display: flex;
+      &-title {
+        font-weight: var(--token-typography-font-weight-semibold);
+        margin-top: sizing.rems(xs);
+        width: 4rem;
```
Comment (cameronperera) [inline ui/admin/app/styles/app.scss:623]:
```suggestion
        width: sizing.rems(xxl) + sizing.rems(m);
```

File: ui/admin/app/styles/app.scss [hunk 4]
```diff
@@ -696,5 +698,5 @@
 
 .target-worker-filter-explainer {
   min-height: 1rem;
```
Comment (cameronperera) [inline ui/admin/app/styles/app.scss:700]:
```suggestion
  min-height: sizing.rems(m);
```

File: ui/admin/app/styles/app.scss [hunk 5]
```diff
@@ -696,5 +698,5 @@
...
-  margin: 1.125rem 0 1.5rem;
+  margin: 1.125rem 0 sizing.rems(l);
```
Comment (cameronperera) [inline ui/admin/app/styles/app.scss:701]:
```suggestion
  margin: (sizing.rems(m) + sizing.rems(xxxs)) 0 sizing.rems(l); // 18 0 24
```

File: ui/admin/app/styles/app.scss [hunk 6]
```diff
@@ -598,69 +598,71 @@
   }
 }
 
-.ordered-series-diagram {
-  --group-title-line-height: 2.25rem;
+.ordered-series-diagram-wrapper {
+  min-height: 150px;
```
Comment (cameronperera) [inline ui/admin/app/styles/app.scss:602]: For these larges ones I think we make exceptions? @randallmorey is that the general rule?

**Review by cameronperera — CHANGES_REQUESTED** [review]

File: ui/admin/app/styles/app.scss (hunk 6 above)
Comment (DhariniJeeva) [reply ui/admin/app/styles/app.scss:602]: I think we can keep all units the same and convert this to rem, 9 or 10 rem could work here

Comment (dvonanderson) [reply ui/admin/app/styles/app.scss:602]: @DhariniJeeva totally agree with you.  There are a few instances in which we use container height and width as px but each fix.

**Review by randallmorey — CHANGES_REQUESTED** [review]
For the min-height fix, I was imagining this going into the target form itself for now.  It needs to contain the migration alert and the diagram description too.  Not sure we know enough about how the order series diagram will be used or evolve to make this kind of change in the component.

**Pull request closed without merging**